src/m3ninx/doc/types.go

```diff
@@ -141,4 +143,26 @@ type OnIndexSeries interface {
 	// Cleanup function must be called once done with the reconciled entry so that
 	// reader and writer counts are accurately updated.
 	ReconciledOnIndexSeries() (OnIndexSeries, resource.SimpleCloser, bool)
+
+	// GetEntryIndexBlockStates returns the entry index block states by time for
+	// the indexed entry.
+	GetEntryIndexBlockStates() EntryIndexBlockStates
```

Comment:
nit: Getter names don't need "Get" prefix as per effective Go?

Reply:
Managed to refactor this out 👍